api/src/main/java/com/discordsrv/api/events/message/preprocess/game/LeaveMessagePreProcessEvent.java

```diff
@@ -44,78 +44,56 @@
  */
 public class LeaveMessagePreProcessEvent extends AbstractGameMessagePreProcessEvent implements PlayerEvent {
 
-    private final DiscordSRVPlayer player;
-    private MinecraftComponent message;
-    private GameChannel gameChannel;
-    private final boolean fakeLeave;
-    private final boolean messageCancelled;
+     private final DiscordSRVPlayer player;
+     private final boolean fakeLeave;
+     private final boolean messageCancelled;
```

Comment:
Please keep indentation to 4 spaces, looks like there's 5 being used in this class